### DISCUSSION TITLE
Ability to list issues in several projects

### DISCUSSION THREAD

**Discussion by ecksun** [ankitpokhrel/jira-cli#182] [Ideas] [answered]
I would like to list issues spanning multiple projects, for example corresponding to this JQL query:

    project in (APP, WEB, AI) AND labels in (backend)

Perhaps by providing a comma separated list to `--project`?

**Comment by ankitpokhrel** [comment]
@ecksun Right now we are binding JQL to a single project context. I think it will be better to provide a way to execute any valid JQL. Since project is a required field what we can do is:
- Check if `project` key exist in provided JQL
- If yes, we will use the query as is
- If not, we will append currently configured project to the query

It should then allow you to use any query combination.

**Comment by ecksun** [comment]
Ah, so that would mean that this would NOT be possible:

    jira issue list --label backend --project APP,WEB,AI

but

    jira issue list --jql 'project in (APP, WEB, AI) AND labels in (backend)'

would?

That works, not as convenient but JQL is not that bad to write

**Comment by wolfiesites** [comment]
IDEA,

Bassicly if u're on support as let's say web dev, working with jira means being in several projects at once. 
Without this feature (to list all issues unrelated to project), for such support person this tool is helpless :( 

it's , I would say, crucial feature to be able to list all issues unrelated to project assigned to "me".

Project its amazing tho, and powerful!

**Comment by ankitpokhrel** [comment, marked as answer]
@ecksun @wolfiesites This feature is available in the latest beta release [v1.0.0-beta.2](https://github.com/ankitpokhrel/jira-cli/releases/tag/v1.0.0-beta.2). Details https://github.com/ankitpokhrel/jira-cli/pull/395.

**Reply by ecksun** [reply]
Looks cool!

I won't be able to test it anytime soon though as I won't be using JIRA for another couple of months for unrelated reasons :(

**Reply by binarious** [reply]
@ankitpokhrel This works great, thank you. Currently we are unable to use that for sprints, right?
```
jira sprint list -q'project in (FOO, BAR)'  
✗ No result found for given query in project "OTHER"
```

**Reply by ankitpokhrel** [reply]
Hi @binarious, yes it won't work with sprint at the moment because it only shows issues from the configured board at the moment. Follow #429 for the fix.
